### PR TITLE
Restored multipath

### PR DESCRIPTION
**Pull request by martinduke**

### CODE REVIEW

File: congestion-control-charter.txt [hunk 1]
```diff
@@ -67,10 +70,9 @@ signals, including specification of explicit signals.
 * Progression of existing Informational or Experimental RFCs to higher maturity,
 if they meet the criteria.
 
-* Operational guidance, in general, including advice to upper layers such as regarding
-counterproductive application retransmissions and multiple connections to the
-same host.
-
+* Operational guidance, in general, including advice to upper layers or new
+transports. Examples include minimum frequency of feedback, counterproductive
```
Comment (zaheduzzaman) [inline congestion-control-charter.txt]: What is a new transport? it this something in lower layer like L3 or L2 transport or a new L4 transport protocol?

Comment (martinduke) [reply congestion-control-charter.txt]: L4

Comment (zaheduzzaman) [reply congestion-control-charter.txt]: lets make it clear then - s/transports/transport protocols.

File: congestion-control-charter.txt [hunk 2]
```diff
@@ -51,6 +51,9 @@ requires approval of the Responsible AD.
 on an assessment of their maturity and likelihood of near-term wide-scale
 deployment.
 
+* Multipath congestion control that detects when two paths share the same
+bottleneck.
```
Comment (renghardt) [inline congestion-control-charter.txt]: I might be missing context here, but: Can we be more generic here and simply say "Multipath congestion control"? I'm not sure if the detection in a mandatory component of such an algorithm, but also I'm happy to be corrected.

**Review by renghardt — COMMENTED** [review]
Will this bit make CONGRESS "The Multipath WG"? :smiley:

File: congestion-control-charter.txt (hunk 2 above)
Comment (martinduke) [reply congestion-control-charter.txt]: I think that's the actual problem to solve, but if that explanatory text troubles you I can delete it.

**Review by renghardt — APPROVED** [review]

**Pull request merged**